@xen-orchestra/web/src/components/vm/VmResource.vue

```diff
@@ -0,0 +1,60 @@
+<template>
+  <UiCard class="vm-resource">
+    <UiTitle>
+      {{ $t('resource-management') }}
+    </UiTitle>
+    <template v-for="(value, key) in generalInfo" :key>
+      <VtsQuickInfoRow :label="$t(key)" :value="String(value)" />
+    </template>
+  </UiCard>
+</template>
+
+<script setup lang="ts">
+import type { XoVm } from '@/types/xo/vm.type'
+import VtsQuickInfoRow from '@core/components/quick-info-row/VtsQuickInfoRow.vue'
+import UiCard from '@core/components/ui/card/UiCard.vue'
+import UiTitle from '@core/components/ui/title/UiTitle.vue'
+import { formatSizeRaw } from '@core/utils/size.util'
+import { useI18n } from 'vue-i18n'
+
+type GeneralInfo = {
+  'cpu-mask': string
+  'cpu-weight': number
+  'cpu-cap': number
+  'minimum-cpu-limit': string
+  'maximum-cpu-limit': string
+  'vm-limit-topology': string
+  'minimum-static-memory': string
+  'maximum-static-memory': string
+  'minimum-dynamic-memory': string
+  'maximum-dynamic-memory': string
+  gpus: string
+}
+
+const { vm } = defineProps<{ vm: XoVm }>()
+const { t } = useI18n()
+
+const staticMinMemoryFormated = formatSizeRaw(vm.memory.static[0], 0)
```

Comment:
Here, if the `vm` prop is updated, the memory will not be re-computed.
To avoid this issue, you can use a `computed` (you can apply the same on the other variables below):
```suggestion
const staticMinMemoryFormated = computed(() => formatSizeRaw(vm.memory.static[0], 0))
```